package.json

```diff
@@ -73,7 +73,6 @@
     "react-helmet": "^5.2.0",
     "react-redux": "^5.0.6",
     "react-router-bootstrap": "0.24.4",
-    "react-router-dom": "^4.2.2",
```

Comment:
Shouldn't it suppose to be a `devDependency` now as it is still needed for testing/building?